### DISCUSSION TITLE
Build Audacity with own PortAudio fork

### DISCUSSION THREAD

**Discussion by aholzinger** [audacity/audacity#6624] [Development]
Hi,
what is the right way to build Audacity with an own (github hosted) PortAudio fork?

Background:
I want to add a new PA host API to PA and make it available on Audacity. When CMake is running it's pulling PortAudio from Conan and I didn't manage to find out where I would have to specify "my own" PA fork (obviously also CMake based) to be compiled together with Audacity.

**Comment by LWinterberg** [comment]
You can disable conan and use your system libraries as described here: https://github.com/audacity/audacity/blob/master/BUILDING.md#disabling-conan

**Comment by aholzinger** [comment]
Hi Leo, thank you.
well, I didn't want to provide **all** libraries, I "just" wanted to provide PA only.
